docs/docs/configuration/live.md

```diff
@@ -21,7 +21,14 @@ webRTC works by creating a websocket connection on extra ports. One of the follo
 * Frigate is run with `network_mode: host` to support automatic UDP port pass through locally and remotely. See https://github.com/AlexxIT/go2rtc#module-webrtc for more details
 * Frigate is run with `network_mode: bridge` and has:
     * Router setup to forward port `8555` to port `8555` on the Frigate device.
-    * For local webRTC, you will need to create your own go2rtc config:
+    * Added port to your docker ports list, for example:
+    ```yaml
+        ports:
+      - "5000:5000" # Frigate Web UI
+      - "8555:8555" # go2rtc
```

Comment:
This needs to be updated like the installation docs to pass both 8555 tcp and 8555 udp. Also probably needs to be formatted.